src/main/java/net/neoforged/neoforge/transfer/storage/TransferAction.java

```diff
@@ -0,0 +1,27 @@
+package net.neoforged.neoforge.transfer.storage;
+
+/**
+ * Represents an action that can be taken when transferring resources.
+ * <br>
+ * {@link #SIMULATE} will simulate the action, allowing the caller to determine the outcome without actually performing the action.
+ * <br>
+ * {@link #EXECUTE} will actually perform the action.
+ */
+public enum TransferAction {
```

Comment:
Could we please get two helper methods added similar to [these](https://github.com/mekanism/Mekanism/blob/1446814b1bcd1a2f5dbded5280c5db3327270011/src/api/java/mekanism/api/Action.java#L36-L56)?

Reply:
Marking as resolved since this enum is gone.